### DISCUSSION TITLE
[FEATURE REQUEST] Flair and/or tag profiles

### DISCUSSION THREAD

**Discussion by atomicpoet** [mastodon/mastodon#21450] [General]
Add the ability to flair and tag profiles as they're viewed in stream. This allows you to quickly identify and categorize certain profiles quickly and efficiently. Useful for when you:

1. Notice certain profiles regularly talking about a topic
2. Want color code or track certain profile attributes
3. Easily track accounts you may later block.

**Comment by ThatOneCalculator** [comment]
Would this be federated or be only for the user? If so, where would the tag data be stored in the actor?

**Comment by atomicpoet** [comment]
It would probably be easier to build this for users. Mostly I want to use this feature in a client.

**Reply by rbairwell** [reply]
Yep, sounds like a "only known and displayed by the client" setup for me as well.
Okay, "my" instance will want to offer me somewhere to save and load them from (to support multiple apps/platforms etc consistently) - but apart from that nicety, no server, network or protocol changes are needed.
(Admittedly, 3rd party apps could implement this "today" with syncing via Dropbox or other similar platform)
